webpack/ForemanTasks/Components/TasksTable/TasksTableActions.js

```diff
@@ -94,29 +95,76 @@ export const unselectRow = id => ({
   payload: id,
 });
 
-export const actionSelected = (actionType, selected, url) => async dispatch => {
+export const bulkResumeRequest = resumeTasks => {
+  const ids = resumeTasks.map(task => task.id);
+  const url = new URI('/foreman_tasks/api/tasks/bulk_resume');
+  url.setSearch('task_ids[]', ids);
+  return API.post(url);
+};
+
+export const bulkResume = (selected, currentURL) => async dispatch => {
+  const resumeTasks = selected.filter(task => task.isResumeble);
+  if (resumeTasks.length < selected.length)
+    dispatch(
+      addToast({
+        type: 'warning',
+        message: __('Not all the selected tasks can be resumed'),
```

Comment:
Is it possible to say which tasks cannot be resumed?  Could be frustrating if the user is not informed of this.

Reply:
Tasks that can't be resumed don't have a resume button near them and I think that's enough indication.
This happens if a user selects tasks that don't have a resume button and resumes in a bulk

Reply:
@MariaAga If one or more of the selected tasks is not resumable, couldn't the resume action be disabled?  (Could even change the text or add a tooltip that says "not all selected tasks are resumable/cancelable" or something.)  Better to tell the user before they try than after. 
@Rohoover @waldenraines thoughts?

Reply:
I didn't want to disable it for situations where a user 'selects all' and only a few are not resumable